_integration-schemas/darksky/forecast.md

```diff
@@ -0,0 +1,292 @@
+---
+tap: "darksky"
+version: "1.0"
+
+name: "forecast"
+doc-link: "https://darksky.net/dev/docs#overview"
+singer-schema: "https://github.com/singer-io/tap-darksky/blob/master/tap_darksky/schemas/forecast.json"
+description: "This table contains information about the forcasts."
```

Comment:
While this description is technically correct, I think there's probably more we should include here after reviewing Darksky's API docs. 

For example:
1. The tap only retrieves data for the `daily` and `hourly` data blocks, but there are a bunch of other ones we exclude.
2. A single record in the table always pertains to a given date (`forecast_date`), *and*
3. A single record in the table always pertains to a given location (The `locations` the user enters into Stitch)
4. The units that measurements are returned in is determined by what the user enters into the **Units** field in Stitch. I can see folks writing into support about discrepancies if this isn't clear, so let's repeat it a bunch to try and prevent that.

Let's try to make it obvious to the reader what's available before they even open the table schema:

```suggestion
description: |
  The `forecasts` table contains weather conditions for a particular date and location. The locations are determined by the locations entered into the **Locations** field in Stitch.
  
  **Note**: The units data points are returned in is determined by the value entered into the **Units** field in Stitch. For example: If `us` is entered, data will be returned in Imperial units. 
```